src/TinyIoC/TinyIoC.cs

```diff
@@ -1650,6 +1650,30 @@ public bool Unregister(Type registerType, string name)
             return RemoveRegistration(typeRegistration);
         }
 
+        public void UnregisterMultiple(Type registrationType, IEnumerable<Type> implementationTypes)
+        {
+            if (implementationTypes == null)
+                throw new ArgumentNullException("implementationTypes", "types is null.");
+
+            foreach (var type in implementationTypes)
+//#if NETFX_CORE
+//              if (!registrationType.GetTypeInfo().IsAssignableFrom(type.GetTypeInfo()))
+//#else
+                if (!registrationType.IsAssignableFrom(type))
+//#endif

```

Comment:
I was unsure about including the commented out `#if NETFX_CORE`. This code was copied from the implementation of `public MultiRegisterOptions RegisterMultiple(Type registrationType, IEnumerable<Type> implementationTypes)` so I left it in for consistency.